src/pymodaq_plugins_thorlabs/daq_move_plugins/daq_move_KPZ101.py

```diff
@@ -0,0 +1,109 @@
+from pymodaq.control_modules.move_utility_classes import DAQ_Move_base, main, comon_parameters_fun
+from pymodaq.utils.logger import set_logger, get_module_name
+
+from pymodaq_plugins_thorlabs.hardware.kinesis import serialnumbers_piezo, Piezo
+
+
+logger = set_logger(get_module_name(__file__))
+
+
+class DAQ_Move_KPZ101(DAQ_Move_base):
+    """
+    Wrapper object to access Piezo functionalities, similar to Kinesis instruments 
+    """
+    _controller_units = 'V'
+    _epsilon = 0.0
```

Comment:
do not let a 0.0 epsilon has it will be difficult to reach

Reply:
We fixed this by setting _epsilon to 0.1.